scanner/src/main/kotlin/experimental/ScanController.kt

```diff
@@ -58,6 +58,11 @@ class ScanController(
      */
     private val provenanceResolutionIssues = mutableMapOf<Identifier, MutableList<OrtIssue>>()
 
+    /**
+     * TODO: Docs
```

Comment:
Uhm, shouldn't this have been filled before merging?

Reply:
My bad, I forgot to add the docs -> https://github.com/oss-review-toolkit/ort/pull/5769